command/pr_create.go

```diff
@@ -40,8 +40,8 @@ func computeDefaults(baseRef, headRef string) (defaults, error) {
 		out.Title = utils.Humanize(headRef)
 
 		body := ""
-		for _, c := range commits {
-			body += fmt.Sprintf("- %s\n", c.Title)
+		for i := len(commits) - 1; i >= 0; i-- {
+			body += fmt.Sprintf("- %s\n", commits[i].Title)
```

Comment:
Golang neither provides a reverse iterator, nor a method to reverse a slice :disappointed: 
So had to do it the old-fashioned way

Reply:
Oh right 🤦‍♂️ I've suggested to use `sort.Reverse`, but that probably does more than just reverse the slice

old-fashioned way is good!